### PR TITLE
[BD-46] feat: added missing SCSS variables in design tokens

### LINKED ISSUE #2347: Try to find additional missing SCSS variables in design tokens
Similar to the case of a missing design token for `$rounded-pill`, let's use this issue as a sanity check around trying to find any other potentially design tokens missing for existing SCSS variables.

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
## Description

- Similar to the case of a missing design token for $rounded-pill, let's use this issue as a sanity check around trying to find any other potentially design tokens missing for existing SCSS variables.

Issue: https://github.com/openedx/paragon/issues/2347

## Merge Checklist

* [ ] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Netlify deploy preview, if applicable.
* [x] Does your change adhere to the documented [style conventions](https://github.com/openedx/paragon/blob/master/docs/decisions/0012-css-styling-conventions)?
* [ ] Do any prop types have missing descriptions in the Props API tables in the documentation site (check deploy preview)?
* [ ] Were your changes tested using all available themes (see theme switcher in the header of the deploy preview, under the "Settings" icon)?
* [ ] Were your changes tested in the `example` app?
* [ ] Is there adequate test coverage for your changes?
* [ ] Consider whether this change needs to reviewed/QA'ed for accessibility (a11y). If so, please add `wittjeff` and `adamstankiewicz` as reviewers on this PR.

## Post-merge Checklist

* [ ] Verify your changes were released to [NPM](https://www.npmjs.com/package/@edx/paragon) at the expected version.
* [ ] If you'd like, [share](https://github.com/openedx/paragon/discussions/new?category=show-and-tell) your contribution in [#show-and-tell](https://github.com/openedx/paragon/discussions/categories/show-and-tell).
* [ ] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @PKulkoRaccoonGang!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: styles/css/core/variables.css [hunk 1]
```diff
@@ -32,7 +32,8 @@
   --pgn-elevation-modal-zindex: 1050;
   --pgn-elevation-modal-backdrop-zindex: 1040;
   --pgn-elevation-dropdown-zindex: 1000;
-  --pgn-transition-collapse: height .35s ease;
+  --pgn-transition-collapse-width: width .35s ease;
```
Comment (monteri) [inline styles/css/core/variables.css:35]: Is there reason behind adding `collapse-width`. Don't remember if we use some width collapse.

Comment (PKulkoRaccoonGang) [reply styles/css/core/variables.css:35]: We have a variable for this case, it looks like you added it :)

File: styles/scss/core/bootstrap-override/mixins/_grid-framework.scss [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+// Used only by Bootstrap to generate the correct number of grid classes given
+// any value of `$grid-columns`.
+
+@mixin make-grid-columns($columns: $grid-columns, $gutter: $grid-gutter-width, $breakpoints: $grid-breakpoints) {
+  // Common properties for all breakpoints
+  %grid-column {
+    position: relative;
+    width: 100%;
+    padding-right: calc($gutter * .5);
+    padding-left: calc($gutter * .5);
```
Comment (PKulkoRaccoonGang) [inline styles/scss/core/bootstrap-override/mixins/_grid-framework.scss:12]: In these files, we have small changes

For example `padding-right: $gutter * .5` to `padding-right: calc($gutter * .5)`

**Review by monteri — APPROVED** [review]

**Review by adamstankiewicz — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@PKulkoRaccoonGang 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/paragon&entry.867055334=https://github.com/openedx/paragon/pull/2364&entry.1484655318=https://github.com/PKulkoRaccoonGang&entry.752974735=2023-06-09+08:03&entry.1917517419=2023-06-16+11:49&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**